src/main/java/com/coreos/jetcd/EtcdClient.java

```diff
@@ -14,15 +15,20 @@
     private final ManagedChannel channel;
 
     private final EtcdKV kvClient;
+    private final EtcdLease leaseClient;
 
     public EtcdClient(ManagedChannelBuilder<?> channelBuilder, EtcdClientBuilder builder) {
```

Comment:
@sijie Perhaps we need something like `EtcdClientConfig`, `EtcdClientBuilder` is responsible for generate such config and pass it when construct a real `EtcdClient`.